libs/MobileSync/build.gradle.kts

```diff
@@ -76,3 +76,7 @@ android {
         aidl = true
     }
 }
+
+kotlin {
+    jvmToolchain(17)
```

Comment:
The update to use JVM toolchain was recommended by Gradle, but I only applied it specifically where Gradle suggested it.